scripts/docker/README.md

```diff
@@ -21,8 +21,9 @@ This repo contains the scripts for creating and pushing two docker images:
 - gatkbase -- a basic docker image that we do not expect to change very often.  The GATK4 docker image uses this one (``FROM``)
 - gatk4 -- the official docker image for GATK4.  The instructions in this document pertain to this image, unless otherwise stated.
 
-``scripts/docker/gatkbase/build_docker_base.sh`` is a script to create the gatkbase docker image.
+``scripts/docker/gatkbase/build_docker_base.sh`` is a script to create the gatkbase docker image. 
 ``build_docker.sh`` is a script to create the full gatk4 docker image.
+``build_docker_remote.sh`` is a script to create the full gatk4 docker image using google cloud build remotely. NOTE: this requires the user first specify their project with the command `gcloud config set project <PROJECT>` to a project that has access to google cloud build. 
```

Comment:
Is it worth mentioning somewhere that one potential motivation for using the remote version is if you have M1 hardware?